open_city_profile/settings.py

```diff
@@ -213,3 +214,20 @@
                 "Please create a %s file with random characters to generate your secret key!"
                 % secret_file
             )
+
+# Youth profile settings
+
+GENDERS = (
```

Comment:
IMHO would be cleaner to keep these in `youths/consts.py`, if they are gonna be used only in this app, or in `open_city_profile/consts.py`, if they might be used elsewhere

Reply:
settings usually have more to do with the internal of the project. now that I think about it, I should have put `CONTACT_METHODS` in `consts` too 😅

Reply:
excellent point. only in this app, so I'll make a `consts.py` file.